apps/src/lib/node/ledger/protocol/mod.rs

```diff
@@ -220,6 +220,7 @@ where
             }
             Ok(TxResult::default())
         }
+        ProtocolTxType::ValidatorSetUpdate(_) => Ok(TxResult::default()),
```

Comment:
I guess we don't need to do any derived transaction just yet for the validator set update transaction, like how we need to for the Ethereum events transaction, but this may change now anyway that we aren't using vote extensions